README.md

```diff
@@ -5,7 +5,7 @@
 
 ###GATK 4 (codename Hellbender)
 
-The public parts of the next generation of GATK/Picard methods engine and tools.
+This repository contains the public parts of the next generation of GATK/Picard methods engine and tools). See also [gatk-protected](https://github.com/broadinstitute/gatk-protected) for the parts of the toolkit that are licence-protected.
```

Comment:
license with an 's' in US english

(http://grammarist.com/spelling/licence-license/)